### ISSUE TITLE
React UI: Alert Rule's "for" field not shown

### ISSUE THREAD

**Issue by s3lph** [prometheus/prometheus#7577]
**What did you do?**

Switch to the new React UI, then navigate to the "Alerts" tab or "Status/Rules" tab.

**What did you expect to see?**

I expected the alert rules to show all important fields, including the field `for`, e.g.:
```yaml
alert: SomethingWrong
expr: up < 1
for: 5m
labels:
  severity: critical
```

**What did you see instead? Under which circumstances?**

The field `for` is not shown, neither in the "Alerts" tab nor in the "Status/Rules" tab, e.g.
```yaml
alert: SomethingWrong
expr: up < 1
labels:
  severity: critical
```
The field is shown when using the classic UI.

**Environment**

* System information:

```
Linux 5.4.46-1-lts x86_64
```

* Prometheus version:

```
prometheus, version 2.19.2 (branch: tarball, revision: 2.19.2)
  build user:       someone@builder
  build date:       20200627-14:50:14
  go version:       go1.14.4
```

**Comment by roidelapluie** [comment]
#7605 and #7606 are being worked on.

**Comment by roidelapluie** [comment]
Fixed in #7605 and #7606
